infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -9,19 +9,26 @@ exports.tasks.push({
     'target-roles',
   ],
   run: async () => {
-    let auth = new taskcluster.Auth(taskcluster.fromEnvVars());
-    let randomId = taskcluster.slugid();
-    let roleId = `project:taskcluster:smoketest:${randomId}`;
-    let payload = {
+    const auth = new taskcluster.Auth(taskcluster.fromEnvVars());
+    const randomId = taskcluster.slugid();
+    const roleId = `project:taskcluster:smoketest:${randomId}*`;
```

Comment:
Let's put some punctuation in there, just for readability
```suggestion
    const roleId = `project:taskcluster:smoketest:${randomId}:*`;
```

Reply:
Yes.